tests/rptest/tests/datalake/datalake_e2e_test.py

```diff
@@ -457,8 +457,7 @@ def test_lag_metrics(self, cloud_storage_type):
 
             dl.create_iceberg_enabled_topic(self.topic_name,
                                             partitions=1,
-                                            replicas=3,
-                                            target_lag_ms=10000)
+                                            replicas=3)
```

Comment:
Curious if we should be setting the lag cluster property to something lower for these tests?

Reply:
you mean something lower than 10s by default? Just to be clear create_iceberg_enabled_topic() sets lag to 10s by default if nothing is set, so the callers (on the left side of diff) were not setting any non default.

Reply:
Right but I mean that before translation lag was introduced, translation was controlled by the commit interval which is pretty low in this test suite.

Reply:
oh interesting, I see a lot of tests setting commit interval to 10s and translation was probably running every 3s or so, good catch, let me try it locally.

Reply:
now I remember, 10s is min bound for this config. Thats why I didn't go lower. I know we have that env variable we can set to bypass bound checks for cluster configs but 10s seems to work with most tests. I just tested upgrade test locally, turns out it is running into some flakiness resulting in hitting a backoff, so had to bump wait timeout but otherwise generally all tests seem ok with 10s. I'm keep this one as is.

Reply:
Hmm I actually don't see any tests setting 10s, but there are ones that set 1s and 5s. In any case, it's not a big deal, I just brought it up to consider given it was a spiritual change in the translation interval when introducing the new scheduler